server/sonar-server/src/test/java/org/sonar/server/user/ws/CreateActionTest.java

```diff
@@ -78,7 +79,7 @@
   public void setUp() {
     System2 system2 = new System2();
```

Comment:
this variable is hiding the property you defined. I don't think indexer is using the `System2` instance you expect it to be using